src/PersonObjects/Sleeve/ui/TaskSelector.tsx

```diff
@@ -290,10 +290,16 @@ function getABC(sleeve: Sleeve): [string, string, string] {
 }
 
 export function TaskSelector(props: IProps): React.ReactElement {
-  const abc = getABC(props.sleeve);
-  const [s0, setS0] = useState(abc[0]);
-  const [s1, setS1] = useState(abc[1]);
-  const [s2, setS2] = useState(abc[2]);
+  const [s0, setS0] = useState("Idle");
+  const [s1, setS1] = useState("------");
+  const [s2, setS2] = useState("------");
+
+  useEffect(() => {
```

Comment:
This has gone from subtle to even more subtle, and I'm pretty sure the reason is because it was designed wrong in the first place. The state here is attempting to be stored *both* locally in the S0/S1/S2 variables, and also in the parent via setABC. I'm pretty sure that duplication is leading to some of these bugs; it should only be in the parent.